packages/edit-site/src/components/sidebar/index.js

```diff
@@ -4,32 +4,33 @@
 import { createSlotFill } from '@wordpress/components';
 import { ComplementaryArea } from '@wordpress/interface';
 import { __ } from '@wordpress/i18n';
-import { cog, pencil } from '@wordpress/icons';
+import { cog, typography } from '@wordpress/icons';
+import { GlobalStylesControls } from '@wordpress/global-styles';
 
 const { Slot: InspectorSlot, Fill: InspectorFill } = createSlotFill(
 	'EditSiteSidebarInspector'
 );
 function Sidebar() {
 	return (
-		<>
+		<div className="edit-site-sidebar">
```

Comment:
Instead of adding a wrapper could we pass the className to ComplementaryArea? This would allow us to avoid rendering the wrapper even if no sidebar is open.
Ideally, we would also avoid calling the class "edit-site-sidebar" and would call it "edit-site-complementary-area".

Reply:
done at https://github.com/WordPress/gutenberg/pull/20530/commits/f44c84caf52e0ed19a4724a57e7680cd64af3a36